app/assets/javascripts/dashboard/service_calendar_logic.js.coffee

```diff
@@ -127,3 +127,10 @@ $(document).ready ->
       url: $(this).data('url')
       data:
         page: page
+
+  # PRINT LISTENER BEGIN
+
+  $(document).on 'click', '#print', ->
```

Comment:
This is a globally accessible action, so it should go in dashboard/global.js.coffee. 
Also, there is no need to duplicate in sub_service_requests.js.coffee because both files are being included already, so you are doubling up the listener.